src/plugins/explore/public/components/query_panel/components/editor_stack/shared.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export type LanguageType = 'nl' | 'kv' | 'ppl';
```

Comment:
could we use enum for this? could we also not use acryonyms for `nl` and `kv`? I think every dev who comes into the codebase should know what `ppl` means, but `nl` and `kv` might be difficult